### PR TITLE
Iter13

### PR DESCRIPTION
**Pull request by kripsy**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+build:
+	go build -o ./cmd/shortener/shortener ./cmd/shortener/*.go
```
Comment (arttet) [inline Makefile]: It would be better if you use `bin`

File: descr.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+run with db
```
Comment (arttet) [inline descr.txt]: Please use MD

File: internal/app/db/db.go [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	
+	CREATE TABLE IF NOT EXISTS public.urls
+	(
+		id bigint NOT NULL,
+		original_url text COLLATE pg_catalog."default" NOT NULL,
+		short_url text COLLATE pg_catalog."default" NOT NULL,
+		CONSTRAINT urls_pkey PRIMARY KEY (id)
+	)
+	
+	TABLESPACE pg_default;
```
Comment (arttet) [inline internal/app/db/db.go]: Please don't use the PostgreSQL tables

File: internal/app/db/db.go [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+	return m, nil
+}
+
+func (mdb PostgresDB) CreateTables(ctx context.Context, myLogger *zap.Logger) error {
+
+	query := `-- Table: public.urls
+
+	--DROP TABLE IF EXISTS public.urls;
+	
+	CREATE TABLE IF NOT EXISTS public.urls
```
Comment (arttet) [inline internal/app/db/db.go]: Please add indexes

**Review by arttet — DISMISSED** [review]

File: internal/app/db/db.go (hunk 4 above)
Comment (kripsy) [reply internal/app/db/db.go]: done

File: internal/app/db/db.go (hunk 3 above)
Comment (kripsy) [reply internal/app/db/db.go]: done

File: descr.txt (hunk 2 above)
Comment (kripsy) [reply descr.txt]: done

File: Makefile (hunk 1 above)
Comment (kripsy) [reply Makefile]: done

**Comment by kripsy** [comment]
in next sprints I'll
- change sql/database to pgxpool
- change insert via pgxtool
- add linter
- add db migrations

**Comment by kripsy** [comment]
```
Добавь, пожалуйста, индексы и миграции. Не используй, таблицы Postgres
```
1) add bin directory
2) add indexes
3) add TODO list
4) add migrations
5) not using postgres table :)

File: README.md [hunk 5]
```diff
@@ -30,3 +34,51 @@ git fetch template && git checkout template/main .github
...
+
+```
+go run ./cmd/shortener/main.go -l "Debug" -f "./test.json"
+```
+
+go run ./cmd/shortener/main.go -l "Debug" -d "host=localhost user=urls password=jf6y5Sfnxsu
+R sslmode=disable port=5432"
+```
+
+Пример запуска локальных тестов
```
Comment (arttet) [inline README.md]: `Пример запуска тестов локально`

File: internal/app/db/db.go (hunk 4 above)
Comment (arttet) [reply internal/app/db/db.go]: Why didn't you remove it?

**Review by arttet — APPROVED** [review]
Well done!

**Pull request merged**